include/swift/AST/DiagnosticsSIL.def

```diff
@@ -979,31 +979,38 @@ NOTE(regionbasedisolation_isolated_since_in_same_region_basename, none,
 //
 
 ERROR(regionbasedisolation_named_transfer_yields_race, none,
-      "transferring %0 may cause a data race",
+      "sending %0 risks causing data races",
       (Identifier))
 
 NOTE(regionbasedisolation_named_info_transfer_yields_race, none,
-     "transferring %1 %0 to %2 callee could cause races in between callee %2 and local %3 uses",
+     "sending %1%0 to %2 callee risks causing data races between %2 and local %3 uses",
```

Comment:
This message still says "callee"; please replace it with "method" for now and later on we should use the `DescriptiveDeclKind` or similar.

Reply:
I still think "method" is better than "callee" in the meantime.